README.md

```diff
@@ -24,14 +24,49 @@ listed in these instructions.
    organized.
 3. There should be at least 3 client-side routes using [React
    Router][react-router]. Be sure to include a nav bar or other UI element that
-   allows users to navigate between routes.
+   allows users to navigate between routes. Use RESTful routing convention where
+   applicable. 
+    | Route Name | URL | HTTP Verb | Description |
+    | ---------- | ---------- | ---------- | ---------- |
+    | Index | /movies | GET | Display all movies |
+    | New | /movies/new | GET | Display new movie form |
+    | Create | /movies | POST | Add new movie to db |
+    | Show | /movies/:id | GET | Display a single movie's info |
+    | Edit | /movies/:id/edit | GET | Display form to update specific movie |
+    | Update | /movies/:id | PUT/PATCH | Update a specific movie |
+    | Destroy | /movies/:id | DELETE | Delete a specific movie |
+
 4. Use a `json-server` to create a RESTful API for your backend and make both a
    `GET` and a `POST` request to the json server. Use a form to make your post
    request, specifically a controlled form/component. Additionally, you may
    choose to incorporate data from an external API but it is not required.
    - You should keep your `json-server` data simple: avoid nested data and
      associations. You'll learn how to work with more complex data in the next
      two phases. Focus on the frontend for this project.
+   - Upon return of json from your POST request, a state update by a setState
+     function is required! 
+     ```
+
+     // in App:
+     function addMovie(newMovie){
+      setMovies([...movies, newMovie]) // Updating movies state.
+     }
+
+     //in Form
+     const configObj = {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({title: "Titanic"})
+     }
+
+     fetch('http://localhost:3000/movies', configObj)
+      .then(res => res.json())
+      .then(data => addMovie(data)) //THIS STATE UPDATE IS REQUIRED!!!
+      // clear form
+
+     ```
+     Do NOT rely solely on the useEffect of another component to load your new data
+     with another GET fetch request. 
```

Comment:
I'd like to make this sound a little more instructional/education-y if we can. What about something like: "Remember: responsibility for re-rendering the page to display the updated list of movies should belong to the `addMovie` function; depending on a subsequent action to load the new data is not best practice." 

Feel free to edit if that doesn't fully get at what you're going for here.

Reply:
That sounds awesome! Thank you, Liz!!!